control-plane/api-gateway/common/routefilter.go

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package common
+
+// +kubebuilder:skipversion
```

Comment:
Wondering what this tag is doing here? I don't think anything in this file involves `kubebuilder`?

Reply:
Was an accident, I'll clean it up